ctp2_code/ui/interface/diplomacywindow.cpp

```diff
@@ -591,16 +614,32 @@ void DiplomacyWindow::UpdateProposalList(ctp2_ListBox *propList, bool toPlayer)
 						}
 					}
 				} else {
+
 					sender = player;
 					receiver = i;
+
 					response = &Diplomat::GetDiplomat(receiver).GetResponsePending(player);
+
 					if (*response == Diplomat::s_badResponse)
+
 						prop = &Diplomat::GetDiplomat(player).GetNewProposalPending(receiver);
 					else
+
 						prop = &Diplomat::GetDiplomat(player).GetMyLastNewProposal(receiver);
+
 					if(*prop == Diplomat::s_badNewProposal)
 						continue;
 
+
+
+
+
+
+
+
+
+
+
```

Comment:
why so many blank lines above?

Reply:
Because they were there before https://github.com/civctp2/civctp2/commit/3285b196b128d975bc560a8b731011399050d949 ;-) Sadly the source is full of inappropriate white space. There are some commits that try to remove some, but do not catch all. I just reverted to the former state, so the white space came in again. If preferred, I can add another that removes them but I would not do it in this commit to really just revert that file to the former state. In the end, luckily white space does not introduce bugs in C/C++ ;-)